index.html

```diff
@@ -2579,9 +2588,16 @@
             <code>String</code>, return <code>false</code> and abort these steps.
           </li>
           <li>
+            Let <code>subdomain_pattern</code> be the result of prepending <code>"."</code>
```

Comment:
IMHO it's fine to leave the '_', and you could also use dashes or spaces between, once all of it is wrapped in `<code></code>`